packages/mobile/src/fiatExchanges/ListItem.tsx

```diff
@@ -0,0 +1,34 @@
+import Touchable from '@celo/react-components/components/Touchable'
+import colors from '@celo/react-components/styles/colors.v2'
+import variables from '@celo/react-components/styles/variables'
+import * as React from 'react'
+import { StyleSheet, View } from 'react-native'
+
+interface Props {
+  children: React.ReactNode
+  onPress: () => void
+  disabled?: boolean
+  testID?: string
+}
+
+export default function ListItem({ children, onPress, disabled, testID }: Props) {
```

Comment:
Is this a generic component of just for exchanges?

Reply:
So far, it's only used in the Fiat Exchange screens, but I think we could find similar places later on and create a generic `ListItem` for different use cases (like with checkmark, with value in the right, etc.).